src/styles/components/form-elements/styles.less

```diff
@@ -34,7 +34,7 @@
     }
 
     & + .form-control-inline {
-      margin-left: @base-spacing-unit * 1/2;
+      margin-left: 0;
```

Comment:
Is it possible to just remove these `margin-left` decorations altogether?

Reply:
I am not sure if they aren't overriding some other styles.